### PR TITLE
feature: opentelemetry support for the `gRPC` and `Temporal` plugins

### PR DESCRIPTION
**Pull request by cv65kr**
# Reason for This PR

https://github.com/roadrunner-server/roadrunner/issues/1313

## Description of Changes

- https://github.com/roadrunner-server/otel/pull/40
- https://github.com/roadrunner-server/otel/pull/41
- https://github.com/temporalio/roadrunner-temporal/pull/322
- https://github.com/roadrunner-server/grpc/pull/69

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

### CODE REVIEW

File: schemas/config/2.0.schema.json [hunk 1]
```diff
@@ -461,6 +461,73 @@
...
+                    "default": false
+                },
+                "exporter": {
+                    "description": "Provides functionality to emit telemetry to consumers",
+                    "type": "string",
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "zipkin",
+                            "stdout",
```
Comment (rustatian) [inline schemas/config/2.0.schema.json:485]: Could you please also add `stderr` (I recently added it).

File: schemas/config/2.0.schema.json [hunk 2]
```diff
@@ -461,6 +461,73 @@
...
+                },
+                "custom_url": {
+                    "description": "Used for the http client to override the default URL",
+                    "type": "string",
+                    "default": ""
+                },
+                "endpoint": {
+                    "description": "Consumer's endpoint",
+                    "type": "string",
+                    "default": "localhost:4318"
```
Comment (rustatian) [inline schemas/config/2.0.schema.json]: 127.0.0.1, while `localhost` is also valid, it can be bonded to a different address.

File: .rr.yaml [hunk 3]
```diff
@@ -543,6 +543,47 @@ service:
...
+  service_name: "rr_test"
+
+  # User's service version
+  #
+  # Default: RoadRunner
+  service_version: "1.0.0"
+
+  # Consumer's endpoint
+  #
+  # Default: localhost:4318
```
Comment (rustatian) [inline .rr.yaml]: 127.0.0.1

**Review by rustatian — CHANGES_REQUESTED** [review]

File: .rr.yaml (hunk 3 above)
Comment (cv65kr) [reply .rr.yaml]: @rustatian all updated

File: .rr.yaml [hunk 4]
```diff
@@ -543,6 +543,47 @@ service:
...
+  compress: false
+
+  # Client to send the spans
+  #
+  # Default: http. Possible values: `http`, `grpc`
+  client: http
+
+  # Provides functionality to emit telemetry to consumers
+  #
+  # Default: otlp. Possible values: otlp (used for new_relic, datadog), zipkin or stdout
```
Comment (rustatian) [inline .rr.yaml]: +stderr

Comment (cv65kr) [reply .rr.yaml]: Sorry, missed that one. Updated

**Pull request merged**